openmetadata-ui/src/main/resources/ui/src/components/common/AirflowMessageBanner/AirflowMessageBanner.tsx

```diff
@@ -32,7 +40,16 @@ const AirflowMessageBanner: FC<SpaceProps> = ({ className }) => {
       data-testid="no-airflow-placeholder"
       size={16}>
       <IconRetry className="align-middle" height={24} width={24} />
-      <Typography.Text>{reason}</Typography.Text>
+      <Typography.Text>
+        {docLink ? (
+          <>
+            {reason?.endsWith('.') ? reason?.slice(0, -1) : reason}{' '}
+            <a href={docLink}>{t('label.here-lowercase')}.</a>
```

Comment:
Can we use the `<Link>` here and pass the target="_blank", so it should open in a new tab?